Source/WebKit/Platform/cocoa/WKMaterialHostingSupport.swift

```diff
@@ -91,17 +92,19 @@ private struct MaterialHostingView<P: MaterialHostingProvider>: View {
         }
     }
 
-    init(content: P.Source, materialEffectType: WKHostedMaterialEffectType = .none, cornerRadius: CGFloat = 0) {
+    init(content: P.Source, materialEffectType: WKHostedMaterialEffectType = .none, colorScheme: WKHostedMaterialColorScheme = .light, cornerRadius: CGFloat = 0) {
         self.content = content
         self.materialEffectType = materialEffectType
+        self.colorScheme = colorScheme
         self.cornerRadius = cornerRadius
     }
 
     var body: some View {
         let view = P.view(for: content)
 
         if let effect = MaterialHostingView<P>.resolvedMaterialEffect(for: materialEffectType) {
-            view.materialEffect(effect, in: .rect(cornerRadius: cornerRadius))
+            AnyView(view.materialEffect(effect, in: .rect(cornerRadius: cornerRadius)))
```

Comment:
what happens if you get rid of `AnyView` and instead use https://developer.apple.com/documentation/swiftui/view/preferredcolorscheme(_:) instead of the color scheme environment?

Reply:
Doesn't work either.

Reply:
😭